doc/source/conf.py

```diff
@@ -97,6 +97,10 @@
 
 .. _MySQL: https://www.mysql.com/
 
+.. |Percona Server| replace:: Percona Server
```

Comment:
I still don't see version number change. Did you made the said changes.

Also, please ensure other doc files are checked too.